packages/rn-tester-e2e/tests/screens/components/buttonComponent.screen.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+import {UtilsSingleton as Utils} from '../../helpers/utils';
+
+class ButtonComponentScreen {
+  // reference in the Components list
+  buttonScreenElement: string = Utils.platformSelect({
+    ios: '[label="Button"]',
+    android: '//android.view.ViewGroup/android.widget.TextView[@text="Button"]',
+  });
+
+  // References to elements within the Button Component screen
+  btnSubmitElement: string = Utils.platformSelect({
+    ios: '[label="Press to submit your application!"]',
+    android: '//android.widget.Button[@resource-id="button_default_styling"]',
```

Comment:
Similar to above, we can have something like

```
function androidWidget(type: string, selector: string, id: string): string {
  return `//android.widget.${type}[@${selector}="${id}"]`;
}
```

Reply:
Hi @kelset,
This is fixed in the last MR. Could you please resolve this comment?